src/store/getters.ts

```diff
@@ -1,6 +1,7 @@
 /**
  * exports the list of store getters.
  */
+import _ from 'lodash';
```

Comment:
Je te suggere de faire plutot
```
import fromPairs from 'lodash/fromPairs';
```
https://medium.com/@armno/til-importing-lodash-into-angular-the-better-way-aacbeaa40473#e748